build.psm1

```diff
@@ -635,9 +635,13 @@ Fix steps:
 
     # download modules from powershell gallery.
     #   - PowerShellGet, PackageManagement, Microsoft.PowerShell.Archive
-    if($PSModuleRestore)
-    {
-        Restore-PSModuleToBuild -PublishPath $publishPath -CI:$CI.IsPresent
+    if ($PSModuleRestore) {
+        Restore-PSModuleToBuild -PublishPath $publishPath
```

Comment:
Can we remove the `CI` switch and related code from `Restore-PSModuleToBuild`?

Reply:
Fixed.